projects/humans/c3d/controllers/c3d_viewer/c3d_viewer.py

```diff
@@ -245,46 +267,51 @@ def getPointsList(reader, name):
         action = value[0]
         if action == 'disable':
             for i in range(1, len(value)):
-                pointRepresentations[value[i]]['visible'] = False
-                pointRepresentations[value[i]]['transparency'].setSFFloat(1.0)
+                c3dfile.pointRepresentations[value[i]]['visible'] = False
+                c3dfile.pointRepresentations[value[i]]['transparency'].setSFFloat(1.0)
         elif action == 'enable':
             for i in range(1, len(value)):
-                pointRepresentations[value[i]]['visible'] = True
-                pointRepresentations[value[i]]['transparency'].setSFFloat(0.0)
+                c3dfile.pointRepresentations[value[i]]['visible'] = True
+                c3dfile.pointRepresentations[value[i]]['transparency'].setSFFloat(0.0)
         elif action == 'radius':
             for i in range(2, len(value)):
-                pointRepresentations[value[i]]['radius'].setSFFloat(float(value[1]))
+                c3dfile.pointRepresentations[value[i]]['radius'].setSFFloat(float(value[1]))
         elif action == 'color':
             h = value[1].lstrip('#')
             color = [int(h[i:i + 2], 16) / 255.0 for i in (0, 2, 4)]
             for i in range(2, len(value)):
-                pointRepresentations[value[i]]['color'].setSFColor(color)
+                c3dfile.pointRepresentations[value[i]]['color'].setSFColor(color)
         elif action == 'graphs':
             if value[2] == 'true':
                 enableValueGraphs.append(value[1])
             else:
                 enableValueGraphs.remove(value[1])
         elif action == 'body_transparency':
-            bodyTransparency = float(value[1])
-            bodyTransparencyField.setSFFloat(bodyTransparency)
+            c3dfile.bodyTransparency = float(value[1])
+            c3dfile.bodyTransparencyField.setSFFloat(c3dfile.bodyTransparency)
         elif action == 'speed':
             playbackSpeed = float(value[1])
             offsetTime = supervisor.getTime()
             totalFrameCoutner = 0
+        elif action == 'c3dfile':
+            with open('default.c3d', 'wb') as file:
+                file.write(base64.b64decode(value[1]))
```

Comment:
Will this work in the released package?
We probably don't want to change files in the installation directory.